library/coretests/tests/floats/f128.rs

```diff
@@ -71,23 +60,23 @@ fn test_num_f128() {
 fn test_num_f128_rem() {
     let ten = 10f128;
     let two = 2f128;
-    assert_eq!(ten.rem(two), ten % two);
+    assert_biteq!(ten.rem(two), ten % two);
```

Comment:
Isn't `rem` the same as `%`? I'm a bit confused by the test.^^

Anyway I think this always worked in Miri ever nice basic arithmetic worked, there's no intrinsics involved here.

Reply:
This test came from https://github.com/rust-lang/rust/blob/8afd71079a729ac002298239f5010ccaac5807ca/library/coretests/tests/floats/mod.rs#L19-L35, indeed it is a bit dubious. I don't see anything else that exercises `Rem` but that would probably be something good to have.